packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -1093,6 +1093,23 @@ class WKWebView extends UIView {
     );
   }
 
+  /// Enables debugging of web contents (HTML / CSS / JavaScript) loaded into any WebViews of this application.
```

Comment:
According to https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/. This flag only affects each `WKWebView` separately, right?

> This decision is made for each individual WKWebView and JSContext to prevent unintentionally making it enabled for a view or context you don’t intend to be inspectable.